test/TestGlobalFunctions.cpp

```diff
@@ -0,0 +1,46 @@
+#include "TestGlobalFunctions.h"
+
+
+Eigen::MatrixXd convertMFEMDenseToEigen(DenseMatrix* mat)
```

Comment:
DenseMatrix* should be const DenseMatrix*

Reply:
Changed.